src/core/appinterface.cpp

```diff
@@ -106,3 +106,11 @@ QVariantMap AppInterface::availableLanguages() const
   }
   return languages;
 }
+
+bool AppInterface::isFileExtensionSupported( const QString &filename ) const
+{
+   const QString extension = filename.mid( filename.lastIndexOf( '.' ) + 1 );
```

Comment:
Why not using QFileInfo here getting the suffix like we do in the FileUtils?

Reply:
Addressed.